### PR TITLE
Link hover styling redux

### LINKED ISSUE #240: Speed up all the animations
They all feel just little bit slow, IMO.

Maybe cut the animation time in half?

### PR DESCRIPTION
**Pull request by laurelfulford**
Created a riff on @karmatosed's #341 that limits where the underline appears by default, and addresses some of the formatting issues.

Fixes #240.

### CODE REVIEW

**Review by josephfusco — CHANGES_REQUESTED** [review]
Can you update editor-style as well please to include the links in there? So far this looks good though!

**Comment by melchoyce** [comment]
Seeing some weird artifacting in the RSS widget links: https://cloudup.com/cNfnMxojTZc

Should also get removed from gallery image hovers:

screen shot 2016-10-17 at 4 35 05 pm

Otherwise looking good, though :)

**Comment by laurelfulford** [comment]
@josephfusco Eep! Thanks for the reminder! Will get them added.

@melchoyce Thanks for the check - will investigate the issues & update!

**Comment by laurelfulford** [comment]
Editor styles, galleries and RSS widget weirdness should all be addressed - just let me know if there's anything else!

**Review by josephfusco — APPROVED** [review]

**Comment by josephfusco** [comment]
Just approving the editor styles changes 👍 .
I'll let @melchoyce confirm the widget issue is resolved.

File: style.css [hunk 1]
```diff
@@ -2695,6 +2705,12 @@ object {
 	margin-bottom: 0;
 }
 
+.tiled-gallery a,
```
Comment (davidakennedy) [inline style.css]: These `.tiled-gallery a` styles aren't needed.

**Comment by melchoyce** [comment]
Still getting the weirdness on the RSS widget.

**Pull request merged**